### DISCUSSION TITLE
[Question][Hw5] Description for solutions

### DISCUSSION THREAD

**Discussion by KalasLavas** [prosyslab-classroom/cs348-information-security#718] [Q&A]
Name: Aziz Huseynov

> If you successfully solve the problem, please write `problemN.txt` which describes how you solve the problem.

How detailed should it be? Is 1-2 sentences okay?

**Comment by KAIST-JongchanPark** [comment]
It is okay if you can explain how you solve the problem with 1-2 sentences.
